digital-credentials/support/iframe.html

```diff
@@ -1,27 +1,69 @@
-<!doctype html>
+<!DOCTYPE html>
 <script src="/resources/testdriver.js"></script>
 <script src="/resources/testdriver-vendor.js"></script>
+<body></body>
 <script type="module">
-import { buildValidNavigatorIdentityRequest, requestIdentityWithActivation } from './helper.js';
+  // @ts-check
+  /**
+   *
+   * @typedef {import('./dc-types').ActionType} IframeActionType
+   * @typedef {import('./dc-types').AbortType} AbortType
+   * @typedef {import('./dc-types').EventData} EventData
+   * @typedef {import('./dc-types').PostData} PostData
+   */
 
-// Loading digital-identity-iframe.html in the test will make a digital credential call on load, and
-// trigger a postMessage upon completion.
-//
-// message {
-//   string result: "Pass" | "Fail"
-//   string data: credential.token
-//   string errorType: error.data
-// }
-
-window.onload = async () => {
-  try {
-    let request = buildValidNavigatorIdentityRequest();
-    let credential = await requestIdentityWithActivation(test_driver, request);
-
-    window.top.postMessage({result: "Pass", data: credential.data}, '*');
-  } catch (error) {
-    window.top.postMessage({result: "Fail", errorType: error.name}, '*');
+  /**
+   * @param {MessageEvent<EventData>} event
+   */
+  async function messageListener(event) {
+    /** @type {EventData} */
+    const data = event.data ? { ...event.data } : {};
+    /** @type {PostData} */
```

Comment:
Is PostData the right type of AbortController?

Reply:
oops, no